widget_driver/lib/src/utils/runtime_environment_info.dart

```diff
@@ -0,0 +1,18 @@
+import 'platform_environment.dart';
+
+/// A helper class which gives you info about
+/// the environment for the current runtime.
+class RuntimeEnvironmentInfo {
```

Comment:
I just renamed this class.. I dropped the prefix which was `Flutter`.
So before this class was called `FlutterRuntimeEnvironmentInfo`.
But that was a bit unneccassery.. So lets just call this `RuntimeEnvironmentInfo` :)